discord-scripts/fix-linear-embed.ts

```diff
@@ -195,6 +199,18 @@ async function processLinearEmbeds(
   logger: Log,
   linearClient: LinearClient,
 ) {
+  if (IGNORED_CHANNELS.has(channel.id)) {
```

Comment:
Any chance we could do these by name instead of id? Just feels easier to reason about. Totally fine if not heh.